firstvoices/backend/models/characters.py

```diff
@@ -256,7 +256,7 @@ def default_g2p_config(self):
 
         # Convert from default g2p config json to site-specific yaml
         default_config_str = yaml.dump(
-            AppJson.objects.get(key="default_g2p_config").json
+            AppJson.objects.get(key="default_g2p_config").json, default_style='"'
```

Comment:
Changing to double quoted yaml to avoid issues with sites containing `'` in the title